packages/inter-protocol/src/proposals/addAssetToVault.js

```diff
@@ -183,7 +183,13 @@ export const registerScaledPriceAuthority = async (
     }),
   );
   const { publicFacet } = E.get(
-    E(zoe).startInstance(scaledPriceAuthority, undefined, terms),
+    E(zoe).startInstance(
+      scaledPriceAuthority,
+      undefined,
+      terms,
+      undefined,
+      `scaledPriceAuthority-${keyword}`,
```

Comment:
If hyphens are used to separate components of the vat identifier, it would be good to use a different separator within sub-parts.

Reply:
There's a `/^[A-Za-z0-9._-]+$/.test(name)` constraint on names, which leaves precious few other delimiters. Which do you prefer?

Reply:
Keep in mind that this is just being used as diagnostic hint for humans to look at, not something with deep semantics.

Reply:
Out of those, I would use '.', but if you want to leave them as is, I won't block anything.

Reply:
Note that it would be very simple to change the rules about what characters are allowed in the label string, but we want to be very careful about anything that would have a misleading meaning if it appears in what looks like a shell command line (e.g, no spaces).

Reply:
We also don't know what other contexts these strings might be embedded in in the future, so it pays to be conservative.